Javascript Space Adventure/student_answers_solutions.js

```diff
@@ -54,22 +54,22 @@ function step2() {
   // /* Declare Variable */ = document.getElementById('size-width').value;
   var width = document.getElementById("size-width").value;
 
-  // TODO: Update the width of the astronaut picture with the inputted width
+  // TODO: Update the height of the astronaut picture with the inputted height
   img.height = height;
 
-  // TODO: Update the height of the astronaut picture with the inputted width
+  // TODO: Update the width of the astronaut picture with the inputted width
   img.width = width;
 
   // Challenge TODO: The desired final picture has a height/width ratio of 2/3
   //                  Instead of manually inputting a value for 'Height', calculate a value based on 'Width' and this ratio
-  var calc_width = (height * 2) / 3;
-  img.width = calc_width;
+  var calc_height = (width * 2) / 3;
+  img.height = calc_height;
```

Comment:
Need to also update validation logic in astronaut.js

Reply:
Alternatively, just update the challenge todo instructions to "width/height ratio of 2/3"